doc/ChangeLog

```diff
@@ -1,4 +1,98 @@
 ===============================================================
+Tag name:  ctsm1.0.dev027
+Originator(s):  negins (Negin Sobhani,UCAR/CSEG,303-497-1224)
+Date: Tue Feb 19 12:57:12 MST 2019
+One-line Summary: Non-constant time initialization for soil hydrology types are moved to SoilHydrologyType.F
```

Comment:
Please fix file name here and in ChangeSum

Reply:
Thanks for the comment. `SoilHydrologyType.F` is changed to `SoilHydrologyType.F90` in both `ChangeSum` and`ChangeLog`.